tests/runtime/intptrcast

```diff
@@ -0,0 +1,11 @@
+NAME Sum casted value
+RUN bpftrace -v -e 'k:vfs_read { @=sum(*(uint8*)(reg("bp")-1)); exit();}'
+EXPECT ^@: [0-9]*
```

Comment:
I think this test should be made more specific, it tests whether we generate valid code but not whether the output makes sense. Instead of attaching to a "random" kprobe we could use a uprobe on a function taking an int and read the nth byte of that. That way the EXPECT can be an exact match. Something like:

```
void fn(int a) {
}

int main() {
  fn(0xA055);
}

@= *(uint8*)(reg("bp")-1)
EXPECT ^@: 160
```

Reply:
Indeed. I'll fix it.